cmd/distribute/distribute.go

```diff
@@ -580,3 +554,27 @@ func stringToBytes32(delegateName string) [32]byte {
 	copy(name[:], delegateName)
 	return name
 }
+
+func checkActionReceipt(c iotex.AuthedClient, hash hash.Hash256) error {
+	time.Sleep(5 * time.Second)
```

Comment:
Generally, we prefer "WaitUntil" rather than "Sleep" (Sleep may introduce many intermittent issues which will be hard to debug).